src/pages/organizers/create/OrganizerForm.tsx

```diff
@@ -78,6 +88,9 @@ const OrganizerForm = () => {
   );
 
   const convertOrganizerToFormData = (organizer: Organizer) => {
+    const isCultuurkuurOrganizer = organizer?.hiddenLabels?.includes(
+      CULTUURKUUR_ORGANIZER_LABEL,
+    );
```

Comment:
Would maybe make this a small helper since we need the same constant and check in `AdditionalInformationStep`